morpheus/models/dfencoder/autoencoder.py

```diff
@@ -905,11 +905,11 @@ def _fit_centralized(self, df, epochs=1, val=None, run_validation=False, use_val
 
                     # Early stopping
                     current_net_loss = net_loss
-                    LOG.debug('The Current Net Loss:', current_net_loss)
+                    LOG.debug('The Current Net Loss: {}'.format(current_net_loss))
 
                     if current_net_loss > last_loss:
                         count_es += 1
-                        LOG.debug('Early stop count:', count_es)
+                        LOG.debug('Early stop count: {}'.format(count_es))
```

Comment:
Use lazy formatting for logger messages:
```suggestion
                        LOG.debug('Early stop count: %s', count_es)
```